### PR TITLE
Perceptron Algorithm: A Hands-On Introduction

### PR DESCRIPTION
**Pull request by lalith1403**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      

#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tool:** [Grammarly](https://grammarly.com)
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by lalith1403** [comment]
This fixes #520...

**Comment by hectorkambow** [comment]
Plagiarism check done

File: articles/perceptron-algorithm/index.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+author: lalithnarayan-c
+date: 2020-10-30T00:00:00-13:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/perceptron-algorithm/hero.jpg
+    alt: perceptron algorithm example image
+---
+Perceptrons were one of the first algorithms discovered in the field of AI. It big significance was that it raised the hopes and expectations for the field of neural networks. Inspired by the neurons in the brain, the attempt to create a perceptron succeeded in modeling linear decision boundaries.
```
Comment (rohanreddych) [inline articles/perceptron-algorithm/index.md:17]: >It big significance 

*its

File: articles/perceptron-algorithm/index.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+We will also look at the perceptron's limitations and how it was overcome in the years that followed.
+
+### Goals
+This article will explain what perceptrons are, and we will implement the perceptron model from scratch using Numpy. For a quick refresher on Numpy, refer to this [article](/engineering-education/matplotlib-visualization-python/).
+
+By the end of the article, you will be able to code a perceptron, appreciate the significance of the model and, understand how it helped transform the field of neural networks as we know it.
+
+### Perceptron
+Frank Rosenblatt developed the perceptron in the mid-1950s, which was based on the McCulloch-Pitts model. The McCulloch-Pitts model was proposed by the legendary-duo [Warren Sturgis McCulloch](https://en.wikipedia.org/wiki/Warren_Sturgis_McCulloch) and [Walter Pitts](https://en.wikipedia.org/wiki/Walter_Pitts). Although these models are no longer in use today, they paved the way for research for many years to come.
```
Comment (rohanreddych) [inline articles/perceptron-algorithm/index.md:29]: >based on the McCulloch-Pitts model

Mention what it is or give links.

**Review by rohanreddych — COMMENTED** [review]
I just went through it, LGTM.

**Comment by lalith1403** [comment]
@rohanreddych @hectorkambow I have made the requested changes. Kindly review the article whenever you get a chance. Thank you.

**Comment by hectorkambow** [comment]
👋  Good afternoon Gary. 
Hope your week is starting off well. 
I wanted to ask if you give this "Perceptron Algorithm" article a tech review whenever you have a chance. 
Thank you v much 🚀

**Review by gwritz — APPROVED** [review]

**Pull request merged**